tests/integration/plugins_nodejs/test_nodejs_plugin.py

```diff
@@ -16,9 +16,9 @@
 import os
 
 import testscenarios
+import yaml as yaml_utils
```

Comment:
Would it be a lot more work to change away from the name "yaml_utils" to "yaml" so future confusion is less likely?

Reply:
this one is going away in a new PR. This is still useful when running in-source